app/services/users.js

```diff
@@ -0,0 +1,25 @@
+const models = require('../models/index');
+const errors = require('../errors');
+const signUpValidator = require('../interactors/sign_up_validator');
+const generatePassword = require('../interactors/generate_password');
+
+module.exports.signUp = body =>
+  models.user.findOne({ where: { email: body.email } }).then(user => {
```

Comment:
Don't change it as it does not make too much of a difference, but look up the `findOrCreate()` function. Just so you learn another way fo doing this :)